pkgs/tools/security/dirbuster/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, stdenvNoCC, fetchurl, makeWrapper, copyDesktopItems, makeDesktopItem, binutils, jre, jdk }:
+
+let
+  name = "dirbuster";
+  version = "1.0-RC1";
+
+  binaryDist = stdenvNoCC.mkDerivation rec {
+    pname = "${name}-bin";
+    inherit version;
+
+    src = fetchurl {
+      url = "mirror://sourceforge/dirbuster/DirBuster%20(jar%20%2B%20lists)/${version}/DirBuster-${version}.tar.bz2";
+      sha256 = "1hgbamzlz83qxcl15nic2jr8542dgq0ghsvrqypc4bhsv7a2v0aj";
+    };
+
+    installPhase = ''
+      # Install dependency binaries
+      install -Dt $out/share/java lib/*
+
+      # Install wordlists
+      install -Dt $out/share/${name} *.{txt,jbrofuzz}
+    '';
+  };
+in
+stdenvNoCC.mkDerivation rec {
+  pname = name;
+  inherit version;
+
+  src = fetchurl {
+    url = "mirror://sourceforge/dirbuster/DirBuster%20Source/${version}/DirBuster-${version}-src.tar.bz2";
+    sha256 = "0p210fp6rbfzvf8a03kmcizbg6f22z621808094dgdpl8d1fj578";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.